resources/ctia-default.properties

```diff
@@ -12,25 +12,27 @@ ctia.http.show.protocol=http
 ctia.nrepl.port=3001
 ctia.nrepl.enabled=true
 
-ctia.store.actor=es
-ctia.store.feedback=es
-ctia.store.campaign=es
-ctia.store.coa=es
-ctia.store.exploit-target=es
-ctia.store.identity=es
-ctia.store.incident=es
-ctia.store.indicator=es
-ctia.store.judgement=es
-ctia.store.sighting=es
-ctia.store.ttp=es
+# Use ES for all Stores
+ctia.store.default.type=es
+ctia.store.default.uri=http://localhost:9200
+ctia.store.default.indexname=ctia
+ctia.store.actor.indexname=ctia_actor
+ctia.store.campaign.indexname=ctia_campaign
+ctia.store.coa.indexname=ctia_coa
+ctia.store.exploit-target.indexname=ctia_exploit-target
+ctia.store.feedback.indexname=ctia_feedback
+ctia.store.identity.indexname=ctia_entity
+ctia.store.incident.indexname=ctia_incident
+ctia.store.indicator.indexname=ctia_indicator
+ctia.store.judgement.indexname=ctia_judgement
+ctia.store.sighting.indexname=ctia_sighting
+ctia.store.ttp.indexname=ctia_ttp
```

Comment:
Thinking about your proposal and the chat with Craig, I think your proposal would be more in line with the possibility of instantiating multiple store instances per entity, is it something we want?